streebog/src/streebog.rs

```diff
@@ -1,6 +1,6 @@
-#![cfg_attr(feature = "cargo-clippy", allow(needless_range_loop, inline_always))]
+#![cfg_attr(feature = "cargo-clippy", allow(clippy::needless_range_loop, clippy::inline_always))]
 use digest::{Input, BlockInput, FixedOutput, Reset};
-use digest::generic_array::typenum::{Unsigned, U64};
+use digest::generic_array::typenum::U64;
```

Comment:
The build fails because you've removed `Unsigned` here, I am not sure why it happens, but until we bump MSRV lets keep it as-is.

Reply:
You're right.
I've fixed it.